queries.yaml

```diff
@@ -177,6 +177,23 @@ post_process:
         14: { area:  100, volume: 200000 }
         15: { area:  100, volume: 100000 }
 
+  - fn: vectordatasource.transform.build_fence
+    params:
+      base_layer: landuse
+      start_zoom: 16
+      prop_transform:
+        kind: true
+        id: true
+        source: true
+        area: true
+        min_zoom: true
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: landuse
+      start_zoom: 16
+      #end_zoom: 20
+      properties: [barrier]
+
```

Comment:
`barrier = props.pop('barrier', None)` drops the barrier tag.

If we updated the `start_zoom` to 0 or whenever landuse started, then yes it would make a difference. But as it stands now it's configured for z16 anyway.